skunkworks_crow/src/main/res/values/strings.xml

```diff
@@ -139,15 +144,14 @@
     <string name="location_permission_needed">Location Permission Needed</string>
     <string name="location_settings_dialog">Enable location from the settings.</string>
 
+    <string name="switch_method">Switch</string>
     <string name="btn_refresh">Refresh</string>
-    <string name="method_bluetooth">Bluetooth</string>
-    <string name="method_wifi_hotspot">Hotspot</string>
-    <string name="title_send_options">Send Options</string>
-    <string name="title_receive_options">Receive Options</string>
     <string name="connect_bluetooth_title">Device to Receive</string>
     <string name="connected_bluetooth_downloading">Connected, start downloading&#8230;</string>
     <string name="connecting_title">Connecting</string>
     <string name="scanning_title">Scanning</string>
+    <string name="bluetooth">bluetooth</string>
+    <string name="hotspot">@string/default_hotspot_ssid</string>
```

Comment:
Why created different string resource? You can use `@string/default_hotspot_ssid` directly, right?

Reply:
Yes, I mean this will make the string name more make sense, right?

Reply:
Fixed